Packs/Infoblox/Integrations/Infoblox/Infoblox.py

```diff
@@ -328,6 +445,19 @@ def parse_demisto_exception(error: DemistoException, field_in_error: str = 'text
     return DemistoException(err_msg)
 
 
+def results_to_context_data(results: Union[Dict, List], to_list: bool) -> Union[Dict[str, Any], List[Dict[str, Any]]]:
+    if to_list:
```

Comment:
This parameter is redundant. You can check the type of `results` using `isinstance(results, list)`.
A better approach:
```python
if not isinstance(results, list):
    results = [results]

context_data = []
for result in results:
    fixed_keys_obj = {RESPONSE_TRANSLATION_DICTIONARY.get(key, string_to_context_key(key)): val 
        for key, val in result.items()}
    context_data.append(fixed_keys_obj)

return context_data
```